src/net/sourceforge/kolmafia/session/ChoiceControl.java

```diff
@@ -4742,6 +4757,27 @@ public static void postChoice2(final String urlString, final GenericRequest requ
         }
         break;
 
+      case 276:
+        // The Gong Has Been Bung
+        String form =
+            switch (ChoiceManager.lastDecision) {
+              case 1 -> "Roach";
+              case 2 -> "Mole";
+              case 3 -> "Bird";
+              default -> "";
+            };
+
+        // We are now in a pseudo LimitMode
+        Preferences.setString("currentLlamaForm", form);
+        KoLCharacter.setLimitMode(LimitMode.NONE);
```

Comment:
a comment saying that this has the side effect of setting the actual limit mode would be appreciated

Reply:
Done